app/src/main/java/org/fossasia/openevent/activities/LocationActivtiy.java

```diff
@@ -68,9 +87,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         getMenuInflater().inflate(R.menu.menu_speakers_activity, menu);
-        SearchView searchView =
+        searchView =
                 (SearchView) menu.findItem(R.id.search_sessions).getActionView();
         searchView.setOnQueryTextListener(this);
+        if (searchText!=null) {
+            searchView.setQuery(searchText, false);
+        }
+        super.onCreateOptionsMenu(menu);
```

Comment:
shouldn't you wait for the super method to apply the styles etc before setting the query back?